src/remote-config.ts

```diff
@@ -90,6 +91,7 @@ export const defaultRemoteConfig: RemoteConfig = {
   enable_car_sharing_in_map: false,
   enable_geofencing_zones: false,
   enable_vehicle_operator_logo: false,
+  minimum_app_version: '1.52.0',
```

Comment:
It's best to set it `undefined` by default I think (although the current value also works)

Reply:
It complained about setting it to undefined. So changed to empty string.